src/themes/default/templates/core/nav.html

```diff
@@ -120,10 +120,9 @@
                     &nbsp;</i>Register</a></li>
             {% endif %}
         </ul>
-
-        <form method="POST" action="/search/" class="form-inline mt-2 mt-md-0">
+        <form method="POST" action="{{ request.journal_base_url }}{% url 'search' %}" class="form-inline mt-2 mt-md-0">
```

Comment:
We use `{% journal_url 'search' %}` for absolute journal URLs or just django's default `{% url 'search' %}` for the relative URL. This is to avoid discrepancies between `path` and `domain` modes